src/main/resources/templates/start.html

```diff
@@ -28,31 +28,29 @@ <h2>Add or View Collections</h2>
 
         </div>
     </div>
-    <p style="padding-bottom:1%"></p>
 
-    <div style="background-color:#0082C4">
-    <div class="container" style="background-color:#0082C4">
-        <h2>Browse Collection</h2>
-        <div class="row row-cols-auto">
-            <div class="col">
-                <div th:replace="fragments/reusable-components :: itemcard">
+    <div class="collection-div">
+        <div class="container collection-div2">
+            <h2>Browse Collection</h2>
+                <div class="row row-cols-auto">
+                <div class="col">
+                    <div th:replace="fragments/reusable-components :: itemcard">
+                    </div>
                 </div>
-            </div>
-            <div class="col">
-                <div th:replace="fragments/reusable-components :: itemcard">
+                <div class="col">
+                    <div th:replace="fragments/reusable-components :: itemcard">
+                    </div>
                 </div>
-            </div>
-            <div class="col">
-                <div th:replace="fragments/reusable-components :: itemcard">
+                <div class="col">
+                    <div th:replace="fragments/reusable-components :: itemcard">
+                    </div>
                 </div>
-            </div>
-            <div class="col">
-                <div th:replace="fragments/reusable-components :: itemcard">
+                <div class="col">
+                    <div th:replace="fragments/reusable-components :: itemcard">
+                    </div>
                 </div>
             </div>
         </div>
-        <p style="padding:1%"></p>
-    </div>
```

Comment:
In-line styling can create technical debt, as we will need to go and upgrade each invidual tag in order to make changes the the CSS of the page. Instead, we should use CSS files for our CSS code. This will save us time and energy as we will not have to search for where those individual tags.

The in-line styling was also creating issues as there were empty <p> tags that were merely being used to create spacing from the divs. Either padding needs to be used on the divs themselves, or we should be looking to use the <br> tag as a last resort. It is usually considered best practice to rely on CSS over HTML to control the presentation of the page.

Citations: [https://www.lifewire.com/avoid-inline-styles-for-css-3466846](https://www.lifewire.com/avoid-inline-styles-for-css-3466846)